### DISCUSSION TITLE
Should cache pattern apply to `path` not `data`?

### DISCUSSION THREAD

**Discussion by Stephen-Gates** [frictionlessdata/frictionlessdata.io#645] [Q&A]
In the [caching of resources pattern](https://frictionlessdata.io/specs/patterns/#caching-of-resources) it states

> Implementations `MAY` handle a `_cache` property on any descriptor that supports a `data` property.

I think `data`  needs to be replaced with `path` based on [data location](https://frictionlessdata.io/specs/data-resource/#data-location) in the Data Resource spec.

**Comment by zaneselvans** [comment]
I was just looking over this too and had exactly the same concern. The `data` property is supposed to be for inline data that's stored inside the `datapackage.json` file itself, but [all of the `_cache` examples](https://specs.frictionlessdata.io/patterns/#specification-2) are making references to remote URLs as you'd expect for `path`.

**Comment by rufuspollock** [comment]
Good point and will look into this.
